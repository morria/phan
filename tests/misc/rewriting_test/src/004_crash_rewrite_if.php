<?php
if ("string" = $p2);
