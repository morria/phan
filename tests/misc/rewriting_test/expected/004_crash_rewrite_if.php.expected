src/004_crash_rewrite_if.php:2 PhanInvalidNode Expected left side of assignment to be a variable
src/004_crash_rewrite_if.php:2 PhanSyntaxError syntax error, unexpected '='
src/004_crash_rewrite_if.php:2 PhanUndeclaredVariable Variable $p2 is undeclared
